components/Players/PlayerManager.vue

```diff
@@ -210,6 +210,23 @@ export default Vue.extend({
     }
   },
   created() {
+    addEventListener('keydown', (e: KeyboardEvent) => {
```

Comment:
I haven't removed the event listener since PlayerManager is always present in the default view. If I should remove, and re add it every time the player is minimized/enlarged, let me know.

Reply:
@Camc314 I would so we don't have unexpected thing once this clients gets bigger and bigger.

Home header and #520 swiper uses keyboard presses as well and who knows if they could interfere.